app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
   <%= nav_link 'Applications', applications_path %>
   <%= nav_link 'Activity', activity_path %>
   <%= nav_link 'Archived', archived_applications_path %>
+  <% if current_user.may_deploy? %><%= nav_link 'Site settings', site_path %><% end %>
```

Comment:
Don't know if this needs three blocks, but that is more of a style comment than a required change

```
<%= nav_link('Site settings', site_path) if current_user.may_deploy? %>
```

Reply:
Of course, silly me.  I'll fix this in a rebase and then merge.  Thanks for the review!